app-shopping/src/main/java/com/shopping/service/PhoneServiceImpl.java

```diff
@@ -47,4 +49,23 @@ private PhoneModel updateColor(PhoneModel modle) {
         return modle;
     }
 
+    public Long addPhone(PhoneModel pm)
+    {
+        return modelRepository.save(pm).getId();
+    }
+
+    public void deletePhone(String modelPrice)
```

Comment:
use @Transactional on top of the deletePhone method

Reply:
Thanks for the input, delete is working now.